capplets/display/meson.build

```diff
@@ -10,7 +10,13 @@ sources += gnome.compile_resources(
   export: true
 )
 
-sources += gnome.genmarshal('foo-marshal', prefix : 'foo_marshal', sources : 'foo-marshal.list')
+marshalers = gnome.genmarshal(
+'foo-marshal',
```

Comment:
What is foo-marshal for?

Reply:
No changes are needed here. Previously, meson compilation failed because the generated files were compiled with autotools first. Now compile meson first, then compile autotools.